Docs/source/building/summit.rst

```diff
@@ -106,16 +106,6 @@ We recommend to store the above lines in a file, such as ``$HOME/warpx.profile``
 
    source $HOME/warpx.profile
 
-Optionally, download and build openPMD-api for I/O (only needed if you did not load our module above):
```

Comment:
Hi, and thanks for this update @ax3l !
I was wondering if we ought to (or already) have the same options as for `AMReX` for choosing the openPMD repo (fork) link / branch name. I know `openPMD` evolves quickly so I wonder if it is necessary to coordinate versions with `WarpX` for users that use it as the output/input format.

Reply:
Thanks for  the question Diana.

I update and notify people once a dependency changes, currently we build with 0.12.0+ of openPMD-api, so nothing changes for users.

Users that use CMake do not see a change even if we upgrade the dependency, since we will pull and build the latest version automatically.

If developers like you want to try out an experimental version of openPMD-api (AMReX, PICSAR) or feature branch, both `ccmake .` documents the options with doc string and we have a full documentation of those options added here now, too (table 2 introduced yesterday with #1699): https://warpx.readthedocs.io/en/latest/building/cmake.html#build-test

Reply:
Thanks @ax3l !!
That is perfect.
Sorry I have been trying to get up to date with the latest PRs... but missed that one.

Reply:
No worries, really brand new :)

Reply:
True! :)
Thanks @ax3l